src/components/ItemUnsupported.tsx

```diff
@@ -0,0 +1,28 @@
+import { useTranslation } from 'react-i18next';
+import { StyleSheet, View } from 'react-native';
+import { Text } from 'react-native-elements';
+
+const ItemUnsupported = () => {
+  const { t } = useTranslation();
+
+  return (
+    <View style={styles.container}>
+      <Text style={styles.text}>
+        {t('This file cannot be opened in the mobile app')}
```

Comment:
Maybe display the item type here so we know what we need to support. 
I think `file` may not be the right wording. I would suggest using `element` instead, as they can be interactive elements (h5p, etherpads etc) and not strictly files.